src/main.c

```diff
@@ -96,6 +92,53 @@ void SystemClock_Config(void)
     while(1);
   }
 }
+
+#endif //STM32F4
+#ifdef STM32F4

```

Comment:
To to jest w ogóle barbarzyństwo i nie ma prawa bytu w mainie. Albo w takiej sytuacji trzeba wyciągnąć funkcję SystemClock _config do osobnego pliku, albo usunąć F4 - pamiętaj, że F4 jest tylko Twoją, developerską wersją. Kod piszemy pod F103, bo on jest na pcb